Added element screenshot functionality

jevgenijusmarinuskinas opened the pull request.

Martynienas commented:
Refactor  test "should capture screenshot of specific page element" under VisualRegression.test.js to use the new functionality

Martynienas reviewed (changes requested):
Refactor test "should capture screenshot of specific page element" under VisualRegression.test.js to use the new functionality

jevgenijusmarinuskinas commented:
> Refactor test "should capture screenshot of specific page element" under VisualRegression.test.js to use the new functionality

Fixed, moved and refactored to VisualRegression.test.js
